docs/commands.md

```diff
@@ -497,6 +497,10 @@ Explicitly set `xcconfig` to use in build.
 
 Location for iOS build artifacts. Corresponds to Xcode's `-derivedDataPath`.
 
+#### `--extra-params <string>`
+
+Custom properties that will be passed to `xcodebuild` command.
```

Comment:
Let's add and example. Also "params" or "flags" instead of "properties" would be more suitable. Make sure to sync the description in the code as well

Reply:
Fixed ✅